### DISCUSSION TITLE
Welcome to The Carpentries Workbench Discussions!

### DISCUSSION THREAD

**Discussion by zkamvar** [carpentries/workbench#1] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of The Carpentries Community about [The Carpentries Workbench](https://carpentries.org/blog/2022/01/live-lesson-infrastructure/). We hope that you:
  * ❓ Ask questions you’re wondering about.
  * 💡 Share ideas.
  * 📑 Share tips and tricks for working with the Workbench.
  * 🤩 Share your testimonials!
  * 🔬 Share [Friction Logs](https://www.trychameleon.com/blog/friction-logs).
  * 👐🏼 Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**This repository will be the central place for discussions around The Carpentries Workbench**

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jas58** [comment]
Hello, I'm Jennifer. 

Thanks for the description of Friction Logs.  

I have been making less formal notes-to-self; this structure shows me how to make them more useful.  I'd like to volunteer as a buddy in Zoom (as suggested).

I am in Github rarely, constantly relearning it.  I am new to Carpentries as an instructor (mostly Spreadsheets and OpenRefine).

**Reply by zkamvar** [reply]
:wave: Hi Jennifer! Thank you for introducing yourself and volunteering! Be on the lookout on Monday for an email with more information about what you can expect during beta 😁

**Comment by dafnidrake** [comment]
Hello, 

I am Drake and currently one of the maintainers for Introduction to Geospatial Raster and Vector Data with R lesson. I am just excited to what is up and coming in the Carpentries community and willing to help out any way I can.

**Reply by zkamvar** [reply]
Hi Drake! I hope you're ready, because things are going to get beta.

**Comment by jhlegarreta** [comment]
Hi,
I am Jon Haitz, one of the maintainers of the [Introduction to dMRI](https://github.com/carpentries-incubator/SDC-BIDS-dMRI) Carpentries incubator lesson. Happy to test the transition to the Carpentries Workbench on the lesson.

**Comment by eirini-zormpa** [comment]
hey all 👋 I'm Eirini, a new maintainer for the R for Social Scientists lesson. I was previously also an instructor and helper on the same lesson. Happy to help out in any way I can 😊

**Comment by jessesadler** [comment]
Hello all. I'm Jesse, and like Eirini, a new maintainer for the R for Social Scientists lesson. I am looking forward to helping where possible in the transition to Workbench.

**Comment by juanfung** [comment]
hello! like erini and jesse, i am a maintainer for the R for Social Scientists lesson (since 2018!). i'm also an instructor. very excited for workbench!

**Comment by spholmes** [comment]
Hi, Susan here, I am an instructor, I usually teach Bioinformatics and Bioconductor using R.

**Comment by SaraMorsy** [comment]
Hi everyone,
I am Sara, I am instructor, lesson developer and maintainer of ELIXIR-UK  DaSH courses under supervision of @Krzysztof Poterlowicz. I am currently responsible for transition our lessons into workbench, glad to join the community

**Comment by fiveop** [comment]
Hi,
I am Philipp, a trainer, and I have used the beta to develop a [GitLab-Lesson](https://github.com/zedif/gitlab-novice/). I like the tool and will probably use it for further lessons. I am particularly interested in having it work on GitLab (and GitLab-Pages) as well, since we are using that software at our university (hence the lesson), and I would be help with that.

**Comment by vhmcck** [comment]
Hi,
Valentina here. I recently completed my instructor training, and I am currently in the process of becoming familiar with The Carpentries community. I am interested in getting involved in the development of the workbench and helping however I can. Happy to join the discussion!
